config/prod/prow/config_knative.yaml

```diff
@@ -861,6 +861,26 @@ periodics:
         memory: 12Gi
       limits:
         memory: 16Gi
+  - custom-job: s390x-e2e-tests
+    cron: 0 8 * * *
+    command:
+    - bash
+    args:
+    - -c
+    - >-
+      mkdir -p /root/.kube &&
+      cp /opt/cluster/config /root/.kube/ &&
+      ./test/e2e-tests.sh --run-tests
+    env-vars:
+    - DISABLE_MD_LINTING="1"
+    - KO_FLAGS="--platform=linux/s390x"
+    - DEPLOY_KNATIVE_MONITORING="0"
+    - SYSTEM_NAMESPACE="knative-eventing"
+    - PLATFORM="linux/s390x"
+    - KUBECONFIG="/root/.kube/config"
+    - SSL_CERT_FILE="/opt/cluster/registry.crt"
+    extra_cluster:
```

Comment:
I think this would be clearer if it were called "external_cluster"? what do you think?
I had to reread the information/comments a few times to figure out why it looked like this, but `external_cluster` would be pretty clear to me.

Reply:
yep, `external_cluster` sounds clearer. Thank you for suggestion!